pkgs/tools/misc/betterdiscordctl/default.nix

```
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
```

Would love to have this to be using `stdenvNoCC`
```suggestion
stdenvNoCC.mkDerivation rec {
```

---

Please create a PR with that change and this update.

---

Done, see #129985.